ui/components/multichain/nft-item/nft-item.test.js

```diff
@@ -1,70 +1,49 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import { fireEvent } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
+import mockState from '../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
 import { NftItem } from '.';
 
 describe('NftItem component', () => {
-  const mockOnClick = jest.fn();
-
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
-
-  it('renders correctly with an image source', () => {
-    const { getByTestId } = render(
-      <NftItem
-        alt="Test Alt"
-        backgroundColor="red"
-        name="Test NFT"
-        src="test-src"
-        networkName="Test Network"
-        networkSrc="test-network-src"
-        tokenId="1"
-        onClick={mockOnClick}
```

Comment:
Nit: Updated the test to Use `renderWithProvider` to avoid rewriting props every time.